eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -4,11 +4,12 @@ steps:
       Write-Host "##vso[task.setvariable variable=X_VCPKG_ASSET_SOURCES_SECRET;issecret=true;]clear;x-azurl,https://cppvcpkgcache.blob.core.windows.net/public-vcpkg-asset-container/,,read"
     displayName: Set Vcpkg Variables
 
-  - task: PowerShell@2
-    inputs:
-      pwsh: true
-      targetType: filePath
-      filePath: eng/scripts/Set-VcpkgWriteModeCache.ps1
-      arguments: -StorageAccountKey '$(cpp-vcpkg-cache-storage-key)'
```

Comment:
@danieljurek can you please make sure the variable group and kv is cleaned up of this value?